src/SprykerEco/Zed/Computop/Business/ComputopFacadeInterface.php

```diff
@@ -15,12 +15,12 @@
 use Generated\Shared\Transfer\PaymentMethodsTransfer;
 use Generated\Shared\Transfer\QuoteTransfer;
 use Generated\Shared\Transfer\SaveOrderTransfer;
-use Spryker\Shared\Kernel\Transfer\TransferInterface;
 
 interface ComputopFacadeInterface
 {
     /**
      * Specification:
+     * - Requires `QuoteTransfer.payment.paymentMethod` to be set for Computop payment.
```

Comment:
as I mentioned above, these facade specification changes should be copied to impacted plugins as well
for this particular case `ComputopSaveOrderPlugin` is affected, but its specification is still not changed

please check all facade methods for this issue

Reply:
Fixed.